_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -11,17 +11,191 @@ redirect_from:
 
 # Segment replication
 
-Segment replication is an experimental feature with OpenSearch 2.3. Therefore, we do not recommend the use of segment replication in a production environment. For updates on the progress of segment replication or if you want leave feedback that could help improve the feature, see the [Segment replication git issue](https://github.com/opensearch-project/OpenSearch/issues/2194). 
-{: .warning}
-
 With segment replication, segment files are copied across shards instead of documents being indexed on each shard copy. This improves indexing throughput and lowers resource utilization at the expense of increased network utilization.
 
-As an experimental feature, segment replication will be behind a feature flag and must be enabled on **each node** of a cluster and pass a new setting during index creation.
-{: .note }
+This is the first step in a series of features designed to decouple reads and writes in order to lower compute costs.
 
-### Potential use cases
+## Use cases
 
 - Users who have high write loads but do not have high search requirements and are comfortable with longer refresh times.
 - Users with very high loads who want to add new nodes, as you do not need to index all nodes when adding a new node to the cluster.
+- Compared to document replication, segment replication performs better in OpenSearch cluster deployments with low replica counts, such as those used for log analytics.
+
+## Segment replication configuration
+
+To set the replication strategy to segment replication, create an index with replication.type set to `SEGMENT`:
+
+````json
+PUT /my-index1
+{
+  "settings": {
+    "index": {
+      "replication.type": "SEGMENT" 
+    }
+  }
+}
+````
+
+## Benchmarking
+
+During experimental testing, our experimental release users reported up to 40% higher throughput with segment replication than with document replication for the same cluster setup.
```

Comment:
Can we validate the 40% call out and what users were working with the experimental release A blog? I think calling out the experimental release leads me to more questions versus just sharing the 40% increase.

```suggestion
During initial benchmarks, segment replication user reported 40% higher throughput with than with document replication for the same cluster setup.
```